src/content/code/plugin-api/layers/override-layer.ts

```
@@ -0,0 +1,68 @@
+  marker: {
+    imageSize: 2,
+    imageColor: "red",
+    labelText: "update",
+  },
+});
+
+// Logging a message to confirm the operation
+console.log("Property override applied to specified layers.");
+
+//2. Define the layer ID and new property value
+const layerId = "01hrv2ddxmg7a64xw9mdq81f6k";
```

Biome

[error] Shouldn't redeclare 'layerId'. Consider to delete it or rename it.

'layerId' is defined here:


(lint/suspicious/noRedeclare)